### PR TITLE
Update identity-enable-qrcodes.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@blowdart  please review and merge to #3795

### CODE REVIEW

File: aspnetcore/security/authentication/identity-enable-qrcodes.md [hunk 1]
```diff
@@ -61,10 +73,16 @@ private string GenerateQrCodeUri(string email, string unformattedKey)
 }
 ```
 
-The second parameter in the call to `string.Format()` is your site name, taken from your solution name. It can be changed to anything value, but it must always be URL encoded.
+The second parameter in the call to `string.Format` is your site name, taken from your solution name. It can be changed to any value, but it must always be URL encoded.
 
 ## Using a different QR Code library
 
-You may have a preferred QR Code library which you want to use. The `qrCode` element in the view is a suitable place to put the QR code.
+You can replace the QR Code library in the `qrCode` element.  <!-- Original
```
Comment (blowdart) [inline aspnetcore/security/authentication/identity-enable-qrcodes.md]: This doesn't have the same meaning. Do you want to change it or shall I after the merge?

Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity-enable-qrcodes.md]: @blowdart  either way. You can use the edit to directly fix it before the merge.

Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity-enable-qrcodes.md]: From the files changes tab, edit and merge directly into this commit. Or fix after merge
image

**Comment by blowdart** [comment]
Done, you can clean up your branch as you wish

**Pull request merged**